# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/security/test/unit/test_csp_upgrade_insecure_request_header.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c dom/security/test/unit/test_csp_upgrade_insecure_request_header.js
--- a/dom/security/test/unit/test_csp_upgrade_insecure_request_header.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/dom/security/test/unit/test_csp_upgrade_insecure_request_header.js	Tue Feb 12 16:08:25 2019 +0000
@@ -87,7 +87,7 @@
     return;
   }
   channel = setupChannel(curTest.contentType);
-  channel.asyncOpen2(new ChannelListener());
+  channel.asyncOpen(new ChannelListener());
 }
 
 function run_test() {